examples/swr-site/pages/remote/graphql-eslint/[[...slug]].mdx

```diff
@@ -14,20 +14,20 @@ export const getStaticProps = async ({ params }) => {
     slug: params.slug
   })
 
-  const content = await fs.readFile(filePath, 'utf8')
-  return {
-    props: {
-      ...(await buildDynamicMDX(content, {
-        defaultShowCopyCode: true,
-        remarkLinkRewriteOptions: {
-          pattern: /^\/docs(\/.*)?$/,
-          replace: '/docs/2.0.0$1'
-        }
-      })),
-      ...(await buildDynamicMeta())
-    },
-    revalidate: 10
-  }
+const content = await fs.readFile(filePath, 'utf8')
+return {
+props: {
+...(await buildDynamicMDX(content, {
+defaultShowCopyCode: true,
+remarkLinkRewriteOptions: {
+pattern: /^\/docs(\/.\*)?$/,
+replace: '/docs/2.0.0$1'
+}
+})),
+...(await buildDynamicMeta())
+},
+revalidate: 10
+}
```

Comment:
seems you have a prettier that runs while committing files, could you rebase your PR once I will merge this PR https://github.com/shuding/nextra/pull/1867 ? So in the future, we'll avoid it since I prettified all repo 😀